cli/cmd/util.go

```diff
@@ -108,3 +112,21 @@ func getKubernetesInterface() (*kubernetes.Clientset, error) {
 
 	return kubeClient, nil
 }
+
+func PromptYesNo(stdin io.Reader, question string) (bool, error) {
+	fmt.Print(question)
+	reader := bufio.NewReader(stdin)
+	response, err := reader.ReadString('\n')
+	if err != nil {
+		return false, err
+	}
+
+	response = strings.ToLower(strings.TrimSpace(response))
+
+	if response != "y" && response != "yes" {
+		fmt.Println("Aborting.")
```

Comment:
We shouldn't print this here. It limits situations in which this function can be used.